web_switch_company_warning/static/src/js/switch_company_warning_worker.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
```

Comment:
[W7903(javascript-lint), ] Use the function form of 'use strict'. [Error/strict]